### DISCUSSION TITLE
Skia project CoreApplication.Exit warning

### DISCUSSION THREAD

**Discussion by mcNets** [unoplatform/uno#10404] [General]
Exit method is working now, it was solved in this PR https://github.com/unoplatform/uno/pull/10001

But compiler still throwing a warning:

`warning Uno0001: Microsoft.UI.Xaml.Application.Exit() is not implemented in Uno`

**Comment by jeromelaban** [comment]
Let's track in https://github.com/unoplatform/uno/issues/10436. Note that the warning you get is for `Application.Exit()`, not `CoreApplication.Exit()`.
